app/src/main/java/mod/hey/studios/project/proguard/ProguardHandler.java

```diff
@@ -209,8 +237,13 @@ public void setProguardFMLibs(ArrayList<String> fullModeLibs) {
 
     public void start(BuildProgressReceiver progressReceiver, Dp dp) throws IOException {
         if (isProguardEnabled()) {
-            progressReceiver.onProgress("ProGuarding classes...");
-            dp.runProguard();
+            if (isR8Enabled()) {
```

Comment:
it might be better to rename `isProguardEnabled()` to `isShrinkingEnabled()` now that it can also mean R8 is used instead of ProGuard

Reply:
done